File: system_template/test/data/system_template/test_export_once/_map.py
[
    # This system tests the "export_once" feature. The repeated export should
    # only be exported once. The behavior uses type_family, which would be
    # duplicated if the export_once feature was not working.
    {
        "source": "test_export_once.behavior.json",
        "target": AUTO_FLAT,
    },
    {
        "source": "test_export_once_extension.behavior.json",
        "target": "BP/entities/test_export_once.behavior.json",
        "on_conflict": "merge",
        "export_once": True
    },
    {
        "source": "test_export_once_extension.behavior.json",
        "target": "BP/entities/test_export_once.behavior.json",
        "on_conflict": "merge",
        "export_once": True
    },
    {
        "source": "test_export_once_extension.behavior.json",
        "target": "BP/entities/test_export_once.behavior.json",
        "on_conflict": "merge",
        "export_once": True
    },
    {
        "source": "test_export_once_extension.behavior.json",
        "target": "BP/entities/test_export_once.behavior.json",
        "on_conflict": "merge",
        "export_once": True
    }
]

File: modular_mc/test/packs/data/modular_mc/auto_map/_map.ts
export const MAP = [
	// No extension replacement
	{
		source: "subdir/auto_map_1.behavior.json",
		target: ":autoFlat",
	},
	{
		source: "subdir/auto_map_1.behavior.json",
		target: ":auto",
	},
	{
		source: "auto_map_2.behavior.json",
		target: ":auto",
	},
	// Extension replacement
	{
		source: "subdir/auto_map_3.feature.json",
		target: ":autoFlat",
	},
	{
		source: "subdir/auto_map_3.feature.json",
		target: ":auto",
	},
	{
		source: "auto_map_4.feature.json",
		target: ":auto",
	},
];
